liquibase-core/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -1484,4 +1491,14 @@ private void addChangeSetMdcProperties() {
         Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_LABEL, labelMdc);
         Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_CONTEXT, contextsMdc);
     }
+
+    private void addSqlMdc(Change change, Database database, boolean generateRollbackStatements) throws RollbackImpossibleException {
+        if (change.supports(database)) {
```

Comment:
What would happen if change doesn't support MDC? Can you also add a javadoc comment to the method?

Reply:
If the change doesn't support the database then we do not add it to the mdc, as we're going to skip it in execution.